doc/reflection-milestone4.md

```diff
@@ -0,0 +1,40 @@
+# Reflection
+
+## What has been implemented
+
+The python dashboard features 7 plots which describe the distribution of data science salaries for the features country, gender, education level, age, tenure, and a scatter plot of all the data science salaries. The dashboard is split into two parts: the left side shows country-specific salary distributions while the right side (marked by a darker background) shows the global salary trends. The global salary trends are colored based on the data scientist's tenure, and interactivity is provided to filter based on tenure in the python dashboard. However, this feature was removed in the R dashboard due to incompatibility of the two plots in plotly. Both functions are fully functional and have been implemented in Heroku.
+
+The user is able to select a specific country by using a dropdown menu (top left) and country-specific plots will be updated to reflect salaries corresponding to the chosen country. A slider is also provided to shift the focus of the country-specific plots depending on what salary range the user is interested in exploring. For instance, the user may be very new to data science, and wants to visualize the lower end of the salary distribution to see what proportion of data scientist's have a bachelor's degree.
+
+## Improvements over previous versions
+
+The main theme in the feedbacks recieved from project reviews was about finetuning the details and the design of the dashboard. The relevant ones incorporated in the current iteration for both python and R dashboards are:  
+
+- Sorted highest paying countries based on median to provide an overall sense of higher paying countries.  
+- Replaced `education_level` with `remote_work` in the histogram.
+- Made salary range vertical for heatmap to increase legibility for the plot and separate it visually from the rest of the plots.  
```

Comment:
Put salary range slider below the dropdown menu to make it clearer that it affects all country-specific plots

Reply:
Oops just saw that you have this as the last bullet point. In that case, delete this bullet point.